mantidproject/mantid
# ALFView Vanadium Normalisation

**Description of work.**
This PR adds the ability to specify an optional vanadium run used for normalisation on the ALFView interface.

**To test:**
Open the ALFView interface
Load ALF82301 as the sample
Load ALF82301 as the vanadium
The resulting InstrumentView plot should look blank, and the ALFData workspace should contain all 1's
Close the ALFView interface and reopen it. The same vanadium run 82301 should be populated automatically

Fixes #34846





---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
